site/schemas/social-media.schema.ts

```diff
@@ -51,11 +50,11 @@ export const refineImageSchema = socialMediaPostSchema
         path: ["hootsuiteProfile"],
         message: "Select a Hootsuite profile",
     })
-    .refine((item) => (item.accountType === "Hootsuite" ? !!item.publishDate : true), {
+    .refine((item) => (item.accountType === "Hootsuite" && !!item.publishTime ? !!item.publishDate : true), {
```

Comment:
As publish date and time are no longer mandatory - is this step required?

Reply:
yes if they do enter it - it needs to be checked for validity